### PR TITLE
Update New-ADOrganizationalUnit.md

### PR DESCRIPTION
**Pull request by baardhermansen**
# PR Summary

Edited content to be the same as other versions of the document set.
See also #3640 

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2019-ps/activedirectory/New-ADOrganizationalUnit.md [hunk 1]
```diff
@@ -42,26 +43,30 @@ You can override property values from the template by setting cmdlet parameters.
 For more information, see the *Instance* parameter description for this cmdlet.
 
 Method 3: Use the **Import-Csv** cmdlet with the **New-ADOrganizationalUnit** cmdlet to create multiple Active Directory OU objects.
-To do this, use the **Import-Csv** cmdlet to create the custom objects from a comma-separated value (CSV) file that contains a list of object properties.
+To do this, use the [**Import-Csv**](https://learn.microsoft.com/powershell/module/microsoft.powershell.utility/import-csv) cmdlet to create the custom objects
```
Comment (get-itips) [inline docset/winserver2019-ps/activedirectory/New-ADOrganizationalUnit.md:46]:
```suggestion
To do this, use the [**Import-Csv**](/powershell/module/microsoft.powershell.utility/import-csv) cmdlet to create the custom objects 
```

File: docset/winserver2019-ps/activedirectory/New-ADOrganizationalUnit.md [hunk 2]
```diff
@@ -161,14 +172,15 @@ Accept wildcard characters: False
...
 Specifies the user account credentials to use to perform this task.
 The default credentials are the credentials of the currently logged on user unless the cmdlet is run from an Active Directory PowerShell provider drive.
 If the cmdlet is run from such a provider drive, the account associated with the drive is the default.
 
-To specify this parameter, you can type a user name, such as User1 or Domain01\User01 or you can specify a **PSCredential** object.
+To specify this parameter, you can type a user name, such as `User1` or `Domain01\User01` or you can specify a **PSCredential** object.
 If you specify a user name for this parameter, the cmdlet prompts for a password.
 
-You can also create a **PSCredential** object by using a script or by using the **Get-Credential** cmdlet.
+You can also create a **PSCredential** object by using a script or by using the [Get-Credential](https://learn.microsoft.com/powershell/module/microsoft.powershell.security/get-credential?view=powershell-7.3) cmdlet.
```
Comment (get-itips) [inline docset/winserver2019-ps/activedirectory/New-ADOrganizationalUnit.md:183]:
```suggestion
You can also create a **PSCredential** object by using a script or by using the [Get-Credential](/powershell/module/microsoft.powershell.security/get-credential?view=powershell-7.3) cmdlet.
```

**Review by get-itips — COMMENTED** [review]
@baardhermansen Thanks for your contributions, please review my comments and apply them to this and other PRs for different OS versions of the article

**Comment by baardhermansen** [comment]
Closing this PR, incorporated into PR #3640

**Pull request closed without merging**